### PR TITLE
modify table

### PR DESCRIPTION
**Pull request by yingjieMiao**
@leogrim

### CODE REVIEW

File: kifi-backend/modules/common/conf/evolutions/shoebox/83.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+alter TABLE failed_content_check alter column mapped_url_hash
+	rename to url2_hash;
+
+alter TABLE failed_content_check alter column prep_url
+	rename to url1;
+
+alter TABLE failed_content_check alter column mapped_url
+	rename to url2;
+
+insert into evolutions (name, description) values('83.sql', 'modify table and column names in failed_uri_normalization');
```
Comment (leogrim) [inline kifi-backend/modules/common/conf/evolutions/shoebox/83.sql:36]: Isn't the code duplicated?

File: kifi-backend/modules/shoebox/app/com/keepit/model/FailedContentCheckRepo.scala [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  def getByUrls(url1: String, url2: String)(implicit session: RSession): Option[FailedContentCheck] = {
+    val (sorted1, sorted2) = sortUrls(url1, url2)
+    val (hash1, hash2) = (NormalizedURI.hashUrl(sorted1), NormalizedURI.hashUrl(sorted2))
+    (for( r <- table if (r.url1Hash === hash1 && r.url2Hash === hash2)) yield r).firstOption
+  }
+
+  def createOrIncrease(url1: String, url2: String)(implicit session: RWSession): Unit = {
+    val (sorted1, sorted2) = sortUrls(url1, url2)
+    val (hash1, hash2) = (NormalizedURI.hashUrl(sorted1), NormalizedURI.hashUrl(sorted2))
+    val r = (for( r <- table if (r.url1Hash === hash1 && r.url2Hash === hash2) ) yield r).firstOption
```
Comment (leogrim) [inline kifi-backend/modules/shoebox/app/com/keepit/model/FailedContentCheckRepo.scala:44]: You could call and match on getByUrls here, and only sort / hash explicitly in the None case

**Comment by leogrim** [comment]
See couple of comments, them good to go. Thanks it's awesome!

File: kifi-backend/modules/common/conf/evolutions/shoebox/83.sql (hunk 1 above)
Comment (yingjieMiao) [inline kifi-backend/modules/common/conf/evolutions/shoebox/83.sql:36]: the first part was commented out. It's just convenient so later I can copy it into SQL. I guess that's a convention.

File: kifi-backend/modules/shoebox/app/com/keepit/model/FailedContentCheckRepo.scala (hunk 2 above)
Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/model/FailedContentCheckRepo.scala:44]: Does that mean I have to write the db calls twice?

File: kifi-backend/modules/common/conf/evolutions/shoebox/83.sql (hunk 1 above)
Comment (leogrim) [inline kifi-backend/modules/common/conf/evolutions/shoebox/83.sql:36]: ok!

**Pull request merged**